network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -575,10 +575,10 @@ def get_context(self, request, *args, **kwargs):
         context['categories'] = BuyersGuideProductCategory.objects.filter(hidden=False)
         context['mediaUrl'] = settings.CLOUDINARY_URL if settings.USE_CLOUDINARY else settings.MEDIA_URL
         # context['coralTalkServerUrl'] = settings.CORAL_TALK_SERVER_URL
-        context['pageTitle'] = f'''{pgettext(
-          'This can be localized. This is a reference to the “*batteries not included” mention on toys.',
-          '*privacy not included'
-        )} - {self.title}'''
+        context['pageTitle'] = f'{self.title} | ' + pgettext(
+                'This can be localized. This is a reference to the “*batteries not included” mention on toys.',
+                'Privacy & Security Guide'
+            ) + f' | Mozilla Foundation'
```

Comment:
when you're using f strings, you don't want to also be using string concatenation:
```suggestion
        context['pageTitle'] = f'{self.title} | {pgettext(
                'This can be localized. This is a reference to the “*batteries not included” mention on toys.',
                'Privacy & Security Guide'
            )} | Mozilla Foundation'
```

Reply:
kk I will update. I was just following the existing code we have elsewhere.

Reply:
there's a bunch of regular strings where we do this, which can only put values "in" the string by concatenating several strings, but when using f strings you get the `{...}` syntax to do that, instead.

Reply:
Hmm, reopening this, because it turns out ```f'{self.title} | {gettext("Privacy & security guide")} | Mozilla Foundation'``` doesn’t work: the string is not extracted by `inv makemessages`